jac-uk/admin
# #2091/self assessment document extraction

## What's included?
Updates to the admin assessment methods view to accommodate word count for self assessments. 

Closes #2091

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Create or find an unlock an exercise.
- Select self assessment with competencies as a method.
- Ensure word count box is shown
- Ensure word count is a required field
- Ensure word count is displayed on assessment methods page next to 'self assessments with competencies'

## Risk - how likely is this to impact other areas?
🟠 Medium risk - this does change code that is shared with other areas

## Additional context
![image](https://github.com/jac-uk/admin/assets/44227249/e678ad48-841e-4c5a-a746-bb09ca3c6cfc)

![image](https://github.com/jac-uk/admin/assets/44227249/c2e6692e-c467-40f3-a6ce-76dbb25e7c0e)


---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
